platform/viewer/src/components/ViewportGrid/ViewportGrid.js

```diff
@@ -30,6 +33,12 @@ const ViewportGrid = function(props) {
     return null;
   }
 
+  useEffect(() => {
+    viewportData.forEach(displaySet => {
+      loadAndCacheDerivedDisplaySets(displaySet, studies);
```

Comment:
I think this is the "IO" to load segs as we switch viewports around

Reply:
It is the IO to switch any derived displayset on the study. Note that the only dependency is `@ohif/core`, and if a displaySet has been created by an extension's sopClassHandler, then the `load` callback from the extension will be called from the displaySet.